src/main/java/app/bpartners/geojobs/service/event/AutoTaskStatisticRecomputingSubmittedService.java

```diff
@@ -47,38 +42,33 @@ public AutoTaskStatisticRecomputingSubmittedService(
   @Override
   public void accept(AutoTaskStatisticRecomputingSubmitted event) {
     var jobId = event.getJobId();
-    var lastDurationValue = event.getMaxConsumerBackoffBetweenRetriesDurationValue();
     var attemptNb = event.getAttemptNb();
     var optionalTiling = tilingJobRepository.findById(jobId);
     if (optionalTiling.isEmpty()) {
       var zoneDetectionJob =
           zoneDetectionJobRepository
               .findById(jobId)
               .orElseThrow(() -> new NotFoundException("Job.id=" + jobId + " not found"));
-      processJob(zoneDetectionJob, jobId, lastDurationValue, attemptNb);
+      processJob(zoneDetectionJob, jobId, attemptNb);
     } else {
       var tilingJob = optionalTiling.get();
-      processJob(tilingJob, jobId, lastDurationValue, attemptNb);
+      processJob(tilingJob, jobId, attemptNb);
     }
   }
 
-  private void processJob(Job job, String jobId, long lastDurationValue, int attemptNb) {
+  private void processJob(Job job, String jobId, int attemptNb) {
```

Comment:
attemptNb is not used anymore here.

Remove it here, and every where it is got inside the service.

Note that as explained by Lou, the new backOff is computed automatically inside the Event not here.